File: JavaScript/twoSum.js
var twoSum = function(nums, target) {
        rtype = []
        for (let i = 0; i < nums.length;i++) {
            for (let x = i + 1; x < nums.length;x++) {
                if (nums[i] + nums[x] == target) {
                    rtype.push(i)
                    rtype.push(x)
                    return rtype }
        } }
};

File: twoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        rtype = []
        for x in range(len(nums)-1):
            for y in range(x+1,len(nums)):
                if nums[x] + nums[y] == target:
                    rtype.append(x)
                    rtype.append(y)
                    return rtype